fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/cascades/TypeRepositoryTest.java

```diff
@@ -180,11 +180,13 @@ void createTypeRepositoryFromArrayTypeWorks() {
     void addSameTypeMultipleTimesShouldNotCreateMultipleMessageTypes() {
         final TypeRepository.Builder builder = TypeRepository.newBuilder();
         final Type t = generateRandomStructuredType();
+        System.out.println("t:" + t);
```

Comment:
Please ignore this test for reviews, it fails because I deleted the `Array.defineProtoType` method, and I'd like to make sure the change makes sense before I fix the tests.